genesis/engine/solvers/rigid/rigid_solver_decomp.py

```diff
@@ -41,7 +41,7 @@ def _sanitize_sol_params(
     timeconst, dampratio, dmin, dmax, width, mid, power = sol_params.reshape((-1, 7)).T
     if global_timeconst is not None:
         timeconst[:] = global_timeconst
-    if (timeconst < gs.EPS).any():
+    if (timeconst <= gs.EPS).any():
```

Comment:
It might be unstable / hacky to rely on the logic of checking timeconst == gs.EPS, but I don't have a very strong opinion here.

Reply:
Yes it is. The good thing is that it is only about print a warning a not, but we should do better I agree. Since the tests are not passing I will do better